static/css/style.css

```diff
@@ -45,16 +45,17 @@ body {
 .container .credit {
 	margin: 20px 0;
 }
-
```

Comment:
Just out of curiosity, why did you remove this blank line, but (for example) added one on line 51?
it seems a bit inconsistent :wink: